.github/workflows/ci.yml

```
@@ -24,11 +24,23 @@ jobs:
       - name: "Install dependencies"
         run: pnpm install --frozen-lockfile
 
+      - name: "Generate content token"
+        id: app-token
+        uses: actions/create-github-app-token@v2
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v2". all required inputs are "app-id", "private-key"

(action)